src/rendering/renderers/shared/extract/ExtractSystem.ts

```diff
@@ -98,6 +98,8 @@ export class ExtractSystem implements System
 
         const { format, quality } = options;
 
+        const imageFormat = format === 'png' ? 'image/png' : 'image/jpeg';
```

Comment:
I don't think this is correct. The format option should be mime type not "png". Can you please create a bug for this?

Reply:
![image](https://github.com/pixijs/pixijs/assets/2418199/d4bf284a-849e-407a-9783-81d884095e38)
Actually the argument that's passed in is this

Reply:
The fix here should probably be this instead:

```ts
type Formats = 'image/png' | 'image/jpeg';
```

Reply:
That's fine too, but png and jpg are easier to use

Reply:
If that's the case, then probably a map:
```ts
const imageTypes: Record<Formats, string> = {
  png: 'image/png',
  jpg: 'image/jpeg',
  webp: 'image/webp',
};
```
Personally, I think the mime type is more transparent, though slightly more verbose, with how toDataURL and other canvas methods are invoked.

Reply:
I agree. In fact, I can accept all of them